data/schemas/schemas_openxmlformats_org_wordprocessingml_2006_main.json

```diff
@@ -51361,6 +51312,59 @@
           ]
         }
       ]
+    },
+    {
+      "Name": "w:CT_NumLvl/w:lvlOverride",
+      "ClassName": "LevelOverride",
+      "Summary": "Defines the LevelOverride Class.",
+      "CompositeType": "OneSequence",
+      "BaseClass": "OpenXmlCompositeElement",
+      "Attributes": [
+        {
+          "QName": "w:ilvl",
+          "PropertyName": "LevelIndex",
+          "Type": "Int32Value",
+          "PropertyComments": "Numbering Level ID",
```

Comment:
See above. This is an example of descriptive `PropertyComments`. In the cases above, either the corresponding XML element's local name was used, or no value was provided (i.e., the key was deleted).

Reply:
@ThomasBarnekow we've noticed this in the past as well and have not gotten around to understanding it. The backend processor does all this automatically. So we'll need to take some time to review the code and understand why it changes these. They should not hold up the code release as they don't affect the logic or API surface.

Reply:
@ThomasBarnekow  this answer relates to all your questions about the PropertyComments as they all are automatically generated and we didn't explicitly change anything. We will check on all of the issues.